src/const.ts

```diff
@@ -120,3 +120,26 @@ export const languageToCharType = {
 }
 
 export const languages = ['en', 'tr', 'vi', 'th', 'ko']
+
+export enum DigitalEmojiUnifiedMap {
+  '0⃣️' = '0️⃣',
```

Comment:
Is the square(ZWS) necessary?

Reply:
Yes, read this document https://dasfoundation.larksuite.com/wiki/wikusrYVycLh2DXSjdwLpxjqOfh